docs/coding-guidelines.md

```diff
@@ -149,6 +149,42 @@ For example the following are correct:
 
 1. Adding TODOs is not recommended and should be avoided whenever possible. In cases in which they do get added each TODO needs to be linked to an issue. Use the full url to the GitHub issue.
 
+1. Do not having trailing whitespace at the end of lines.
+
+1. When using `this` or `base` to call other constructors, put the colon and call to the constructor on a new line.
+
+    This is correct:
+    ```cs
+    public MyClass(string arg)
+      : this(arg, arg2: false)
+    {
+       ...
+    }
+    ```
+  
+    This is incorrect:
+    ```cs
+    public MyClass(string arg) : this(arg, arg2: false)
+    {
+      ...
+    }
+    ```
+
+1. Don't inline calls to methods that have more than 1 parameter.
+
+    This is correct:
+    ```cs
+    string value = GetSomeValue(arg1, arg2);
+    DoSomething(value, arg3);
+    ```
+  
+    This is incorrect:
+    ```cs
+    DoSomething(GetSomeValue(arg1, arg2), arg3);
+    ```
+  
+    Consider using temporary variables even when the inline method call takes zero or one arguments, as it helps debugging, in addition to readability.
```

Comment:
I don't know how I feel about this one. 
I don't like optimizing for debugging. 

Specifically, I am also ok with:

    ```cs
    DoSomething(
        GetSomeValue(arg1, arg2), 
        arg3);
    ```

Reply:
From a debugging perspective it's a bit frustrating, because I can't easily see what value is going to be passed into `DoSomeValue` without stepping into it, then needing to run to the end of the method. By using a temporary variable you can just "step over", inspect the return value and therefore quickly determine if the inputs into `DoSomething` look ok or not.

Reply:
I get that, I just don't think it's a strong enough argument to start enforcing a very very particular pattern.

I will accept it if others overwhelmingly agree. 

In this case, I'd be more likely to just ask people to specify a named param...that's what bugs me with the example, not the fact that there's a method invocation.